src/control/lib/control/pool.go

```diff
@@ -222,7 +222,7 @@ type (
 	// PoolCreateResp contains the response from a pool create request.
 	PoolCreateResp struct {
 		UUID      string   `json:"uuid"`
-		Leader    uint32   `json:"svc_ldr"`
+		SvcLdr    uint32   `json:"svc_ldr"`
```

Comment:
I think you could leave this name as-is, since it's already translated to the correct JSON.